crypto/err/openssl.txt

```diff
@@ -348,6 +348,28 @@ CONF_F_NCONF_NEW:111:NCONF_new
 CONF_F_PROCESS_INCLUDE:116:process_include
 CONF_F_SSL_MODULE_INIT:123:ssl_module_init
 CONF_F_STR_COPY:101:str_copy
+CRMF_F_CRMF_POPOSIGNINGKEY_INIT:100:CRMF_poposigningkey_init
+CRMF_F_OSSL_CRMF_CERTID_GEN:101:OSSL_CRMF_CERTID_gen
```

Comment:
the numbers are already in use, is that a problem?

Reply:
openssl.txt needs to be auto-generated based on latest version in master after every re-base.

Reply:
The values are auto-generated and start with `100` for each sub-library.
It is fine that the values overlap with other sub-libs since they are distinguished by the module identifiers like `ERR_LIB_CRMF` and `ERR_LIB_CONF`.